src/components/Buttons/Button.js

```diff
@@ -319,6 +323,15 @@ Button.propTypes = {
 
   /** Snacks theme attributes provided by `Themer` */
   snacksTheme: themePropTypes,
+
+  /**
+    If passed, users will have the ability to render a custom element
+    or React Component in the DOM while leveraging the styling of a
+    Snacks Button (e.g., using react-router's Link)
+  */
+  customElement: PropTypes.node,
```

Comment:
This proptype isn't correct.. this would be `element={<Component />}` rather than `element={Component}`